ROLES.md

```diff
@@ -0,0 +1,51 @@
+# Roles & responsibilities in support of the Australian Government Design System
+
+> This document communicates the expected engagement from any person or organisation who engages with or uses the new, community supported Australian Government Design System
+
+Matt Sawkill from Code for Australia (@sawks on Slack and Github) is the primary contact for any questions relating to the mapping of roles and responsibilities.
+
+<figure>
+    <img src="https://github.com/designsystemau/TSC/blob/main/images/roles-diagram.png" alt="The Australian Government Design System “support ecosystem”" style="width:100%">
+    <figcaption align = "center">The Australian Government Design System “support ecosystem”</figcaption>
+</figure>
+
+## Maintainers (Technical Steering Committee)
+
+* Maintainers are the core community members responsible for ongoing management and decision-making for the Design System
+* Anyone living or working for Australia can become a Maintainer
+* The group of Maintainers forms the Technical Steering Committee (TSC)
+* Membership, responsibilities expectations of the TSC is covered by this [TSC Charter](https://github.com/designsystemau/TSC/blob/main/CHARTER.md) and our [Code of Conduct](https://github.com/designsystemau/TSC/blob/main/CODE-OF-CONDUCT.md)
+* In TSC, Maintainers have equal voting rights, and there is no other hierarchy within this group
+* Effectively they are the product managers and core developers of the Design System, leading the RFC process and facilitating working groups in order to make consensus decisions over technical architecture and future direction
+* The TSC are responsible managing contributions to the Design System made by supporting organisations, Government agencies and individuals in the broader community
+
+## Supporting Organisations
+
+* Supporting Organisations are third parties who play an active and ongoing role in supporting the work of the TSC and the Design System itself
+* These organisations may include non-for-profits and NGOs, commercial vendors and Government organisations (operating in a different context to use of the Design System in delivery which is covered below) with a vested interest in the sustainability of the Design System
```

Comment:
Does this not translate to "Any entity"?

Reply:
Yes, but it does not hurt to be specific.

Reply:
Fair

Reply:
I will include mention in info text